sql/core/src/test/scala/org/apache/spark/sql/execution/PlannerSuite.scala

```diff
@@ -994,6 +994,88 @@ class PlannerSuite extends SharedSparkSession with AdaptiveSparkPlanHelper {
       }
     }
   }
+
+  test("EnsureRequirements.reorder should fallback to the right side HashPartitioning") {
```

Comment:
`PlannerSuite` already has many tests, so how about making a new test suite `EnsureRequirementsSuite` for the two tests below?

Reply:
Done. thanks.